tests/upgrades/test_repository.py

```diff
@@ -157,17 +156,18 @@ def test_pre_scenario_custom_repo_check(self):
 
         product = entities.Product(organization=org).create()
         create_repo(rpm1, FILE_PATH)
-        repo = entities.Repository(product=product.id, url=CUSTOM_REPO).create()
+        repo = entities.Repository(
```

Comment:
You might've missed this conversion?

Reply:
yep, I can include that in this PR, adding now.

Reply:
Since I was updating this, I updated the other functions too. @JacobCallahan  `entities` and `ssh` imports were eliminated from the module :tada: